src/sentry/objectstore/__init__.py

```diff
@@ -40,6 +40,9 @@ def distribution(
 _ATTACHMENTS_CLIENT: Client | None = None
 _ATTACHMENTS_USECASE = Usecase("attachments", expiration_policy=TimeToLive(timedelta(days=30)))
 
+_PREPROD_CLIENT: Client | None = None
```

Comment:
Now that we have multiple use cases, I think we should default to reusing the same client for all of them. Mind updating this so that both attachments and preprod use a single `_OBJECTSTORE_CLIENT` or similar?